dataStore/influxDbUtil/dbUtil.py

```diff
@@ -0,0 +1,26 @@
+"""Core module for interacting with influxDB"""
+
+__author__ = "李旭升 <programmerli@foxmail.com>"
+__copyright__ = "Licensed under GPLv2 or later."
+
+from influxdb import InfluxDBClient
+
+'''
+myInfluxDbClient是自己封装的InfluxDBClient,
+为了插入和读取InfluxDB
+'''
+
+
+class myInfluxDbClient:
+    def __init__(self, influxDBAddress):
```

Comment:
the port, user, password, db should all be parameterized, rather than hard coded here.

we can provide defaults for them though.